python/numpy-in-pytorch-modules.py

```diff
@@ -1,11 +1,16 @@
 import torch.nn as nn
-import numpy
+import numpy as np
```

Comment:
I was told in the other PR that `import numpy` was preferred.

Reply:
https://github.com/trailofbits/semgrep-rules/pull/12#discussion_r1047158663 was about `numpy` in semgrep rule, not in python test.
That is, in semgrep we want to catch all `numpy` package uses, [even if they are aliased/renamed to something like `np`](https://semgrep.dev/docs/writing-rules/pattern-syntax/#imports).

On the other hand, in tests we want to verify our semgrep rule with not-straightforward code, like e.g., we can `import numpy as something`.